src/app.js

```diff
@@ -1,21 +1,64 @@
+/* eslint-disable no-useless-escape */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+
+const PORT = process.env.PORT || 8080;
+
+const mineTipe = {
+  html: 'text/html',
+  css: 'text/css',
+  js: 'text/javascript',
+  xml: 'text/xml',
+  json: 'application/json',
+  pdf: 'application/pdf',
+};
+
+const server = http.createServer((req, res) => {
+  const normalizeUrl = new URL(req.url, `http://${req.headers.host}`);
+  const fileName = normalizeUrl.pathname.slice(1).split('/');
+
+  if (fileName[0] !== 'file') {
+    res.statusCode = 412;
+
+    res.end('The requested url is incorrect. '
+      + 'The request should look like \"/file/<SEARCH_FILE>\"');
+
+    return;
+  }
+
+  if (fileName.length === 1 || (fileName.length === 2 && !fileName[1])) {
+    fileName[1] = 'index.html';
+  }
+
+  const typeFile = fileName[fileName.length - 1].split('.');
```

Comment:
it's better to rename it to fileExtension
```suggestion
  const fileExtension = fileName[fileName.length - 1].split('.');
```